patch/msft-kernel-ipv6-ra-metric-sysctl.patch

```diff
@@ -0,0 +1,194 @@
+commit 7c12a9ecf4e0c782875e399ef43220a7fce2f8ca
+Author: pchaudha <pchaudha@contoso.com>
+Date:   Tue Dec 29 05:05:01 2020 +0000
+
+    ip6 ra sysctl
+
+diff --git a/Documentation/networking/ip-sysctl.txt b/Documentation/networking/ip-sysctl.txt
+index 7eb9366..8f0dfc8 100644
+--- a/Documentation/networking/ip-sysctl.txt
++++ b/Documentation/networking/ip-sysctl.txt
+@@ -1537,6 +1537,12 @@ accept_ra_defrtr - BOOLEAN
+ 	Functional default: enabled if accept_ra is enabled.
+ 			    disabled if accept_ra is disabled.
+ 
++accept_ra_defrtr_metric - INTEGER
++    Metric for default router learned in Router Advertisement.
++    
```

Comment:
The description needs to be extended, and also the default described and explained.

Reply:
done